src/overlays/gamestates/ovl_file_choose/z_file_choose.h

```diff
@@ -17,7 +17,6 @@ typedef struct FileChooseContext {
     /* 0x00248 */ SkyboxContext skyboxCtx;
     /* 0x00470 */ MessageContext msgCtx;
     /* 0x12550 */ Font font;
-    /* 0x242DC */ UNK_TYPE1 unk_242DC[4];
     /* 0x242E0 */ EnvironmentContext envCtx;
```

Comment:
What's causing it to align to 8?

Reply:
`Font` has `u64` members now to ensure alignment on various buffers